skimage/transform/_warps.py

```diff
@@ -18,14 +17,27 @@
 )
 
 
+def _multichannel_default(multichannel, ndim):
+    if multichannel is not None:
+        return multichannel
+    else:
+        warn('The default multichannel argument (None) is deprecated. '
+             'Specifiy True or False explicitly.')
+        # utility for maintaining previous color image default behavior
+        if ndim == 3:
+            return True
+        else:
+            return False
+
+
 def resize(image, output_shape, order=1, mode=None, cval=0, clip=True,
            preserve_range=False):
```

Comment:
I have not added it so far as I am not sure what the logic would be within the function.  As @jni stated, whether or not the last axis would be reshaped is unambiguous from `image.shape` and `output_shape`.  I suppose if `multichannel=True` we could raise an error if `image.shape[-1] != output_shape[-1]`, but I'm not sure what else it could do?